Alpaca.Markets/Interfaces/IHistoricalItemsV2.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Alpaca.Markets
+{
+    /// <summary>
+    /// Encapsulates list of historical aggregates (bars) from Polygon's v2 REST API.
+    /// </summary>
+    /// <typeparam name="TItem">Type of historical items inside this container.</typeparam>
+    public interface IHistoricalItemsV2<out TItem> : IHistoricalItems<TItem>
```

Comment:
Just extend existing IHistoricalItems<TItem> interface instead of creating new one. Adding new fields is backward-compatible change. You can do it without worrying about possible problems with existing client code. And this V2 suffix discloses unnecessary implementation details for client code.